drivers/firmware/imx/ele_base_msg.c

```diff
@@ -147,7 +147,7 @@ int ele_write_fuse(struct device *dev, uint16_t fuse_id, u32 value, bool lock)
 
 	status = RES_STATUS(priv->rx_msg->data[0]);
 	ind = RES_IND(priv->rx_msg->data[0]);
-	if (status != priv->success_tag) {
+	if (status != priv->success_tag && ind != 0x0) {
```

Comment:
0x29 indicates the request has not been successfully proceeded so that this modification is unnecessary.

Reply:
Writing payload 0x0 failes with:
`fsl-se-fw se-fw2: Command Id[214], Status=0x29, Indicator=0x0`
e.g. `tx_msg: 0x17d60306 0x00202760 0x00000000
`
Assume you have a fuse map and you want to write this binary file 4 byte wise. Then there might be 0x0's, but writing 0x0 is not an error as the indicator shows. Or at least the indicator 0x0 is not documented.

Reply:
@AliceGuo9401 Please comment if you are okay with @linkjumper comment and let us know if there is any adjustment before we merge this.

Reply:
Commit is reverted.

Alternatively, we could check at the beginning of ele_write_fuse whether value == 0x0 and return 0 in this case. If you want a revised fix in a new Pull Request, please let me know.

Reply:
No need to make any changes. EdgeLock® secure enclave firmware will fix that in its next release.